src/pages/NewSensor/NewSensorStepTwo.js

```diff
@@ -11,14 +11,46 @@ import { SUSSOL_ORANGE, WHITE } from '../../globalStyles';
 import { buttonStrings, vaccineStrings } from '../../localization';
 import { WizardActions } from '../../actions/WizardActions';
 import { goBack } from '../../navigation/actions';
+import { selectConfigs } from '../../selectors/newSensor';
+import { NewSensorActions } from '../../actions';
 
-export const NewSensorStepTwoComponent = ({ nextTab, previousTab, exit }) => (
+export const NewSensorStepTwoComponent = ({
```

Comment:
The changes to this component are for disconnecting the `breachconfigrow` component from redux and instead, this component subscribes to the changes for the breach configs and creates the dispatchers to pass down the line to the rows.